src/data/courseDependenciesData.js

```diff
@@ -8,8 +8,6 @@ const courseDependencyData = (courseSlug) =>
         type: 'playlist',
         guid: '93bd',
         slug: 'declarative-uis-without-css-with-elm-ui-93bd',
-        customOgImage:
-          'https://res.cloudinary.com/dg3gyk0gu/image/upload/v1615288740/next.egghead.io/resources/declarative-uis-without-css-with-elm-ui/share-card-declarative-uis-without-css-with-elm-ui_2x.png',
         dependencies: {
```

Comment:
remove the need for setting ogImage in course dependencies data now that they are loading from sanity